src/components/Cards/Results.vue

```diff
@@ -0,0 +1,324 @@
+<template>
+  <div>
+    <!-- mobile view -->
+    <div>
+      <div
+        class="md:flex md:flex-col lg:hidden items-center w-full"
+        v-for="biztype in businessTypes"
+        :key="biztype.id"
+      >
+        <div
+          class="
+            rounded-lg
+            flex flex-col
+            md:px-8
+            py-6
+            px-3
+            bg-white
+            shadow-2xl
+            h-auto
+            w-11/12
```

Comment:
what's the difference between line 20 and 21?

Reply:
done